app/brave_settings_strings.grdp

```diff
@@ -138,7 +138,7 @@
     Bookmarks
   </message>
   <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_USE_AUTOCOMPLETE_COMMANDER" desc="The label for settings checkbox controlling whether or not commander suggestions show up in autocomplete">
-    Quick commands
+    Quick Commands
```

Comment:
This string is used in the Settings > Appearance > Show autocomplete suggestions in address bar. We have a bit of inconsistency there with capitalization, but I think `commands` (along with other entries) should not be capitalized there.

Reply:
Hi @mkarolin, added the change, please take a look.